config/cloud_controller.yml

```diff
@@ -89,6 +89,7 @@ logging:
   file: /tmp/cloud_controller.log
   level: debug2
   syslog: vcap.example
+  gdpr_compliant: false
```

Comment:
Making the flag imply that turning it to true makes the cloud controller fully gdpr-compliant worries me a bit.

Reply:
Hm yea i just wanted to have a parameter that can be reused in multiple parts in the logging code for the same purpose. E.g. we also found some emails in the logs which also have to be hidden to comply, but we are looking at one thing at a time currently.

Reply:
Ah and we have it under the logging node on purpose to indicate it is just anonymizing logs so the can be stored safely and still comply to gdpr. In contrast to other parts of the cc (like database user table etc.) there to keep emails and ips in logs for the CC to do its work. For saving personal data in CC for the purpose of providing the service the gdpr does not limit this as long as you tell the customer. So imho its just the logging that currently is not optimal and thus the config parameter is located there :).

Reply:
😊 I totally did not expand the preview to see it was under logging 👍